artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -608,6 +608,14 @@ public void parseMainConfig(final Element e, final Configuration config) throws
 
       config.setNetworkCheckPingCommand(getString(e, "network-check-ping-command", config.getNetworkCheckPingCommand(), Validators.NO_CHECK));
 
+      config.setAnalyzeCritical(getBoolean(e, "analyze-critical", config.isAnalyzeCritical()));
+
+      config.setAnalyzeCriticalTimeout(getLong(e, "analyze-critical-timeout", config.getAnalyzeCriticalTimeout(), Validators.GE_ZERO));
+
+      config.setAnalyzeCriticalCheckPeriod(getLong(e, "analyze-critical-check-period", config.getAnalyzeCriticalCheckPeriod(), Validators.GE_ZERO));
```

Comment:
I didn't want to use null here. On this case, check period=0 would mean not set.